specifications/api/swagger.yaml

```diff
@@ -311,15 +311,9 @@ x-poolMetadataSource: &poolMetadataSource
       * null -> no fetching
       * direct -> direct fetching
       * uri -> use SMASH server
-  oneOf:
-    - type: string
-      enum:
-        - none
-        - direct
-    - title: URI
-      type: string
-      format: uri
-      example: https://smash.cardano-mainnet.iohk.io/
+  type: string
+  pattern: '(null|direct|https?:\/\/[a-zA-Z0-9-_~.]+/?)'
```

Comment:
The oneOf had an overlap, so I think it's better to provide a lax javascript regex.